tests/braille-specs/critical-apparatus.yaml

```diff
@@ -0,0 +1,20 @@
+
+# Yaml Test For Criticle-Apparatus-common-characters
+
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+display: unicode.dis
+table:
+  language: critical-apparatus
+  __assert-match: critical-apparatus.utb
```

Comment:
To test a uti file, we specify the table file path rather than a query, and/or we test the table(s) that include the file. It's always more interesting to test top-level tables because that is the behaviour that users notice. In this case you could perhaps do both. (You can specify multiple `table:` before a `test:`.)

By the way, in which other tables than hbo are you planning to include this one?